UI/src/javascript/tuition-partner.js

```diff
@@ -0,0 +1,109 @@
+"use strict";
+
+import axios from "axios";
+
+const TuitionPartner = (() => {
+  const getSeoUrlFromId = (id) => id && id.replace("shortlist-tpInfo-cb-", "");
+  const getAddToShortlistFetchUrl = (id) =>
+    `/tuition-partner/${id}?handler=AddToShortlist`;
+  const getRemoveFromShortlistFetchUrl = (id) =>
+    `/tuition-partner/${id}?handler=RemoveFromShortlist`;
+
+  const shortlistCheckboxes = document.querySelector(
```

Comment:
Same thing as `search-results.js` about when the DOM is queried. Would be better to query it when `init()` is called rather than when the file is loaded.